app/assets/javascripts/mumuki_laboratory/application/certificate.js

```diff
@@ -0,0 +1,17 @@
+mumuki.load(() => {
+  scaleCertificate();
+  mumuki.resize(scaleCertificate);
+})
+
+function scaleCertificate() {
```

Comment:
Not a big deal, but just be aware that you are placing this function in the global scope, and we tend to avoid that.